portfolio/src/main/webapp/script.js

```diff
@@ -40,9 +41,19 @@ function toggleSort() {
   getComments();
 }
 
+/* Change navbar color after scrolling past the intro */
+function navbarColorScroll() {
+  var introSection = document.getElementById("intro");
+  var navbar = document.getElementById("navbar");
+  if (window.scrollY > (introSection.offsetTop + introSection.offsetHeight - navbar.offsetHeight)) {
```

Comment:
I can't tell right away what condition this is represents. My recommendation would be to assign this to a local variable ```const isScrolledPastNavBar``` (or whatever it is). Then you could also use a ternary ```?``` operator to assign ```backgroundColor``` if you wanted.